### DISCUSSION TITLE
【不具合？】『コメントした記事』のLGTM数が記事のLGTM数になっている

### DISCUSSION THREAD

**Discussion by NurseAngel** [increments/qiita-discussions#215] [要望・アイデア・不具合]
[コメントした記事](https://qiita.com/rana_kualu/comments)に表示されるLGTM数って、
前はコメント自体のLGTM数だったと思うのですが、
これがいつのまにかコメントした記事のLGTM数になっていました。

記憶頼りなので、私の勘違いだったらすみません。

**Comment by tanaka-asami415** [comment]
@NurseAngel 

お問い合わせいただきありがとうございます🙏
こちらおっしゃる通り、記事カードのUI変更で「コメント自体のLGTM数からコメントした記事のLGTM数」に変更されております。
[リリースノート](https://qiita.com/release-notes#%E8%A8%98%E4%BA%8B%E3%82%AB%E3%83%BC%E3%83%89%E3%81%AEui%E3%81%AE%E5%A4%89%E6%9B%B4%E3%82%92%E6%AD%A3%E5%BC%8F%E7%89%88%E3%81%A8%E3%81%97%E3%81%A6%E5%85%AC%E9%96%8B%E3%81%97%E3%81%BE%E3%81%97%E3%81%9F)　に記載できていなかったため、近日中に追記いたします🙏:sweat_drops:
今回お問い合わせいただき気づくことができました。ありがとうございます🙇
